components/brave_wallet/browser/internal/hd_key_unittest.cc

```diff
@@ -28,6 +32,16 @@ std::string GetHexAddr(const HDKey* key) {
   EthAddress addr = EthAddress::FromPublicKey(pubkey_no_header);
   return addr.ToHex();
 }
+
+std::string GetWifCompressedPrivateKey(std::vector<uint8_t> private_key) {
+  private_key.insert(private_key.begin(), 0x80);  // Version Byte.
+  private_key.push_back(0x01);                    // Compression Byte.
+  auto sha256hash = crypto::SHA256Hash(crypto::SHA256Hash(private_key));
+  private_key.insert(private_key.end(), sha256hash.begin(),
+                     sha256hash.begin() + 4);  // Checksum.
+  return Base58Encode(private_key);
+}
+
```

Comment:
This is now needed only for tests to match test vectors. Most likely to be moved to utils in the future